shakenfist/daemons/cleaner.py

```diff
@@ -181,10 +182,19 @@ def run(self):
                     ni['uuid']).info('Hard deleting network interface')
                 db.hard_delete_network_interface(ni['uuid'])
 
-            # Find nodes which have been offline for a long time
             for n in Nodes([node_inactive_states_filter]):
                 age = time.time() - n.last_seen
-                if age > config.NODE_CHECKIN_MAXIMUM * 10:
+
+                # Find nodes which have returned from being missing
```

Comment:
To be honest, I am unsure if this is the right way to do this, or just to set the node state to created every time the node is observed. The second way would involve a much higher etcd write load, but this way feels ... clunky.